android/app/src/main/AndroidManifest.xml

```diff
@@ -7,28 +7,33 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
-
     <application
         android:name=".OpenEventApp"
         android:allowBackup="false"
-        android:largeHeap="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
+        android:largeHeap="true"
```

Comment:
largeHeap is already enabled in the existing Manifest. I haven't added this. Might have come up because of restructuring